packages/toolpad-app/src/utils/paths.ts

```diff
@@ -1,9 +1,16 @@
 import * as path from 'path';
 
+/**
+ * Normalize a file path to be platform-agnostic.
+ */
+export function normalizePath(importPath: string): string {
```

Comment:
"normalize" is a very generic term, perhaps we should call it `asPosixPath` or something?

Reply:
Sounds good, I actually had started with that name but then changed it :)